CHANGELOG.md

```diff
@@ -2,12 +2,20 @@
 
 ## Unreleased
 
+
+## [0.7.7] - [2021-02-09]
+
+### Added
+
 * Added serializer properties when creating an avro serializer
+
 * Exposed deserializer and serializer properties in the serde-resolver. Ideally I would have liked the configs to be restructured, but extended what we have to not break compatibility.
```

Comment:
Hey Tassos, looks good but could you turn the second phrase into a neutral form (not using "I")?

Reply:
Actually removed the second phrase